<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDialog</class>
 <widget class="QDialog" name="PasswordDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>279</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Open Encrypted Volume</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="PushButtonOptions">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Options</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonOpen">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonCancel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxReadOnly">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mount In &amp;Read Only Mode</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonMountPointPath">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>select mount point path</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PushButtonVolumePath">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open volume path</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassPhraseField">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPassPhraseFromFile">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>open key file</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="OpenVolumePath">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="MountPointPath">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassphrase">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelMoutPointPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mount Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolumePath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume Path</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbKeyOption">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>210</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cbKeyType">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Password</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KeyFile</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Key+KeyFile</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Plugin</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="cbShareMountPoint">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Share Mount Point</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbVolumeType">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>LUKS/TrueCrypt/BitLocker</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VeraCrypt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VeraCrypt System</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PLAIN dm-crypt</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelVolumeType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume Type</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlainDmCryptOptions">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLuksExternalHeaderPath">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVolumeProperty">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>301</width>
     <height>29</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelVolumeProperty">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxVisibleKey">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="veraCryptWarning">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>581</width>
     <height>281</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbOK">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>121</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
  </widget>
  <widget class="QFrame" name="frameOptions">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>561</width>
     <height>261</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>129</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Set File System Options</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditFsOptions">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSet">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCancel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>OpenVolumePath</tabstop>
  <tabstop>PushButtonVolumePath</tabstop>
  <tabstop>MountPointPath</tabstop>
  <tabstop>PushButtonMountPointPath</tabstop>
  <tabstop>checkBoxReadOnly</tabstop>
  <tabstop>cbShareMountPoint</tabstop>
  <tabstop>cbVolumeType</tabstop>
  <tabstop>pushButtonPlainDmCryptOptions</tabstop>
  <tabstop>lineEditVolumeProperty</tabstop>
  <tabstop>cbKeyType</tabstop>
  <tabstop>checkBoxVisibleKey</tabstop>
  <tabstop>PassPhraseField</tabstop>
  <tabstop>pbKeyOption</tabstop>
  <tabstop>pushButtonPassPhraseFromFile</tabstop>
  <tabstop>PushButtonOpen</tabstop>
  <tabstop>PushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
